### PR TITLE
Api class structure

### PR DESCRIPTION
**Pull request by cubap**

### CODE REVIEW

**Comment by cubap** [comment]
There's a lot moving here, but let me know if you see anything missing.

File: api/User.mjs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        }
+        return response.json()
+      })
+      .catch((error) => {
+        // Alert user with error message
+        throw error
+      })
+  }
+
+  renderProjects(containerId) {
```
Comment (cubap) [inline api/User.mjs:58]: This doesn't belong here

**Review by thehabes — APPROVED** [review]
Can /utilities/getHash.mjs be removed or deprecated?

I combed through each file for anything obvious.  There could be better documentation in some places now that things have settled in.  Since I was commenting on that I made documentation format and placement consistent, but comment content is still inconsistent.

I have not yet tested the code manually at the time of submitting this but will move into that to double check that there are not any broken imports or failing components.

**Review by git-voo — APPROVED** [review]
Looks great, no broken functionality from the previous point, I'd create new issues from observations here.

**Comment by thehabes** [comment]
> Looks great, no broken functionality from the previous point, I'd create new issues from observations here.

I also did not find any broken functionality when testing the components.

**Pull request merged**